tensorboard/components/tf_ng_tensorboard/types/api.ts

```diff
@@ -55,3 +55,19 @@ export interface Dashboard {
    */
   isReloadDisabled: boolean;
 }
+
+export interface PluginMetadata {
```

Comment:
How do these two new types mesh with `Dashboard` above?

These new types correspond to the backend APIs, and having types for
them makes sense, but are they also going to be the types passed around
in the frontend code? e.g., the naming conventions don’t match up.

Reply:
@nfelt I think you are replying to the older diff. The Dashboard which was a copy over from the Polymer-based TensorBoard is now removed. Just do note that the plan is to use this new type throughout only when we can figure out the build stuff.

Reply:
Ah sorry yeah, I had the older diff still up.